gpu-operator/platform-support.rst

```diff
@@ -309,16 +308,23 @@ The GPU Operator has been validated in the following scenarios:
          -
          -
 
-    .. _ubuntu-kernel:
+    .. _kubernetes-version:
 
     :sup:`1`
-    For Ubuntu 22.04 LTS, kernel versions 6.5 and 5.15 are LTS ESM kernels.
+    The Kubernetes community only supports the last three minor releases. Older releases
```

Comment:
It would be good to point to an official link when stating this

Reply:
We can cite this [page](https://kubernetes.io/releases/) as reference

Reply:
updated!